### ISSUE TITLE
Failing unit test? testEscaping(com.impossibl.postgres.jdbc.DatabaseMetaDataTest)

### ISSUE THREAD

**Issue by dstipp** [impossibl/pgjdbc-ng#217]
Is anyone else seeing this failure?

```
Tests run: 42, Failures: 0, Errors: 1, Skipped: 0, Time elapsed: 2.689 sec <<< FAILURE! - in com.impossibl.postgres.jdbc.DatabaseMetaDataTest
testEscaping(com.impossibl.postgres.jdbc.DatabaseMetaDataTest)  Time elapsed: 0.049 sec  <<< ERROR!
com.impossibl.postgres.jdbc.PGSQLSimpleException: LIKE pattern must not end with escape character
    at com.impossibl.postgres.jdbc.ErrorUtils.makeSQLException(ErrorUtils.java:159)
    at com.impossibl.postgres.jdbc.ErrorUtils.makeSQLException(ErrorUtils.java:136)
    at com.impossibl.postgres.jdbc.PGConnectionImpl.execute(PGConnectionImpl.java:442)
    at com.impossibl.postgres.jdbc.PGStatement.executeStatement(PGStatement.java:375)
    at com.impossibl.postgres.jdbc.PGPreparedStatement.execute(PGPreparedStatement.java:267)
    at com.impossibl.postgres.jdbc.PGPreparedStatement.executeQuery(PGPreparedStatement.java:283)
    at com.impossibl.postgres.jdbc.PGDatabaseMetaData.execForResultSet(PGDatabaseMetaData.java:112)
    at com.impossibl.postgres.jdbc.PGDatabaseMetaData.getTables(PGDatabaseMetaData.java:1047)
    at com.impossibl.postgres.jdbc.DatabaseMetaDataTest.testEscaping(DatabaseMetaDataTest.java:800)
```

I appear to get it on JDK 7 & 8 on PostgreSQL 9.1 only (passes on 9.2, 9.3, 9.4) and wanted to check if this was isolated to my setup or if others see this as well. (Found in the Travis CI builds running 'mvn -e test' against 441b120)

**Comment by kdubb** [comment]
This has always failed on 9.1 but since we  (certainly I) stopped testing on 9.1 a while ago it was forgotten.  This is something that needs to be fixed for 9.1

Kudos for the travis test against server versions.  Love it.

**Comment by kdubb** [comment]
Fixed by #256
